packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -24,6 +24,11 @@ export class QuoteRequestor {
         takerApiKey: string,
         takerAddress: string,
     ): Promise<SignedOrder[]> {
+        if (marketOperation !== MarketOperation.Sell) {
```

Comment:
This was helpful so I didn't have to deal with a null `takerAmount` param in the tests.  I think this should be done regardless though.